### PR TITLE
Adding PMFBY branch

### PR DESCRIPTION
**Pull request by naanrdk**

### CODE REVIEW

File: projects/cpi/data/external/State_LGD_CPI_Inflation.csv [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Andhra Pradesh,28
+Arunachal Pradesh,12
+Assam,18
+Bihar,10
+Chandigarh,4
+Chhattisgarh,22
+Dadra And Nagar Haveli,26
+Daman And Diu,25
+Delhi,7
+Goa,30
```
Comment (Suryakandukuri) [inline projects/cpi/data/external/State_LGD_CPI_Inflation.csv:13]: Please remove this file from this branch. we dont keep data files on git

File: projects/cpi/src/data/process_cpi_data.py [hunk 2]
```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+from datetime import datetime
+from pathlib import Path
+
```
Comment (Suryakandukuri) [inline projects/cpi/src/data/process_cpi_data.py:7]: @gursharan-info  please cross check if the script is fine.

File: projects/cpi/src/data/get_inflation_data.py [hunk 3]
```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import time
+from datetime import datetime
+from pathlib import Path
```
Comment (Suryakandukuri) [inline projects/cpi/src/data/get_inflation_data.py:7]: @gursharan-info  please cross check if the script is fine.

File: projects/cpi/src/data/get_cpi_data.py [hunk 4]
```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import time
+from datetime import datetime
+from pathlib import Path
+
```
Comment (Suryakandukuri) [inline projects/cpi/src/data/get_cpi_data.py:8]: @gursharan-info  please cross check if the script is fine.

File: projects/fertilizer-sales/pyproject.toml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+authors = ["Agnidhra Ghosh"]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+python-dotenv = "^0.13.0"
+click = "^7.1.2"
+pandas = "^1.0.5"
+numpy = "^1.19.0"
+nbconvert = "^6.2.0"
+
```
Comment (Suryakandukuri) [inline projects/fertilizer-sales/pyproject.toml:14]: necessary packages are missing here @AgnidhraG for pdf parsing as well as the scraping. please check

File: projects/fertilizer-sales/src/data/get_raw_data.py [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+        #   continue
+        Year = Select(driver.find_element_by_id("finYear"))
+        Year.select_by_index(i)
+        State = Select(driver.find_element_by_id("stateName"))
+        State.select_by_index(j)
+        print("State:", j)
+        submit = driver.find_element_by_id("submit")
+        submit.click()
+        time.sleep(3)
+        try:
```
Comment (Suryakandukuri) [inline projects/fertilizer-sales/src/data/get_raw_data.py:40]: @AgnidhraG , where are we storing the data through this script?

File: projects/pm_kisan/pyproject.toml [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+description = "PM Kisan Data"
+authors = ["Agnidhra Ghosh"]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+python-dotenv = "^0.13.0"
+click = "^7.1.2"
+pandas = "^1.0.5"
+numpy = "^1.19.0"
+
```
Comment (Suryakandukuri) [inline projects/pm_kisan/pyproject.toml:13]: necessary packages are missing. please add them via poetry add <packagename>. @AgnidhraG

File: projects/pm_kisan/src/data/get_raw_data.py [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+        "InstallmentNumber": "installment_number",
+        "MaleBeneficiary": "male_beneficiary",
+        "FemaleBeneficiary": "female_beneficiary",
+        "General_Beneficiary": "general_beneficiary",
+        "SC_Beneficiary": "sc_beneficiary",
+        "ST_Beneficiary": "st_beneficiary",
+        "FTOGeneratedRate": "fto_generated_rate",
+    },
+    inplace=True,
+)
```
Comment (Suryakandukuri) [inline projects/pm_kisan/src/data/get_raw_data.py:113]: please comment the code. @AgnidhraG

File: projects/primary-census-abstract/pyproject.toml [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+description = "This project contains scraping and processing scripts for PCA demography and PCA Household Amenities data"
+authors = ["Agnidhra Ghosh"]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+python-dotenv = "^0.13.0"
+click = "^7.1.2"
+pandas = "^1.0.5"
+numpy = "^1.19.0"
+
```
Comment (Suryakandukuri) [inline projects/primary-census-abstract/pyproject.toml:13]: @AgnidhraG please add necessary packages through poetry

**Review by Suryakandukuri — CHANGES_REQUESTED** [review]

File: projects/cpi/data/external/State_LGD_CPI_Inflation.csv (hunk 1 above)
Comment (gursharan-info) [reply projects/cpi/data/external/State_LGD_CPI_Inflation.csv:13]: Aren't external files supposed to be like helper files required for that particular dataset?  
If we remove it, then where would this file be available when anyone else has to work and process this dataset?

**Comment by Suryakandukuri** [comment]
wrong PR

**Pull request closed without merging**